process/mock/vmContainerMock.go

```diff
@@ -12,59 +12,56 @@ type VMContainerMock struct {
 	KeysCalled        func() [][]byte
 }
 
-func (V *VMContainerMock) Get(key []byte) (vmcommon.VMExecutionHandler, error) {
-	if V.GetCalled == nil {
+func (v *VMContainerMock) Get(key []byte) (vmcommon.VMExecutionHandler, error) {
+	if v.GetCalled == nil {
 		return &VMExecutionHandlerStub{}, nil
 	}
-	return V.GetCalled(key)
+	return v.GetCalled(key)
 }
 
-func (V *VMContainerMock) Add(key []byte, val vmcommon.VMExecutionHandler) error {
-	if V.AddCalled == nil {
+func (v *VMContainerMock) Add(key []byte, val vmcommon.VMExecutionHandler) error {
+	if v.AddCalled == nil {
 		return nil
 	}
-	return V.AddCalled(key, val)
+	return v.AddCalled(key, val)
 }
 
-func (V *VMContainerMock) AddMultiple(keys [][]byte, vms []vmcommon.VMExecutionHandler) error {
-	if V.AddMultipleCalled == nil {
+func (v *VMContainerMock) AddMultiple(keys [][]byte, vms []vmcommon.VMExecutionHandler) error {
+	if v.AddMultipleCalled == nil {
 		return nil
 	}
-	return V.AddMultipleCalled(keys, vms)
+	return v.AddMultipleCalled(keys, vms)
 }
 
-func (V *VMContainerMock) Replace(key []byte, val vmcommon.VMExecutionHandler) error {
-	if V.ReplaceCalled == nil {
+func (v *VMContainerMock) Replace(key []byte, val vmcommon.VMExecutionHandler) error {
+	if v.ReplaceCalled == nil {
```

Comment:
maybe you can refactor here... if v.ReplaceCalled !=nil

Reply:
no need for this